parsers/cobol.c

```diff
@@ -16,7 +16,15 @@
 #include "parse.h"
 #include "routines.h"
 
-#define COBOL_REGEX_PREFIX "^.......[ \t]*"
+/*
+ * ANS85 can have letters in the first column
+ */
+#define COBOL_REGEX_PREFIX_FIXED "[0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z][ \t]+"
+
+/*
+ * Sourceformat free/variable just needs the line to start with optional spaces/tabs
+*/
+#define COBOL_REGEX_PREFIX_RELAXED "[ \t]+"
```

Comment:
How about introducing following definition?
```
#define COBOL_REGEX_PREFIX_UNIFIED "([0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z][0-9a-Z])?[ \t]+"
```
So we can make cobolTagRegexTable simple.